test/README.md

```diff
@@ -52,29 +53,19 @@ pyenv local cr  # auto-activate virtualenv when entering this directory
 make init-dev
 ```
 
+Common Issues
+--------------
 
-Set up Python 3.4 using virtualenv (outdated!)
-----------------
-```bash
-# Install Python 3.4
-brew update
-brew install python3  # Note! This is now out of date and will install a later version of python. :(
-python3 -V  # should be 3.4.x
+1. `cx_freeze` installation failure on OSX:
 
-# Install pip, virtualenv, virtualenvwrapper if you don't have them
-wget https://bootstrap.pypa.io/get-pip.py
-python get-pip.py
-pip install virtualenv
-pip install virtualenvwrapper
+    If you are using a Mac for dev setup, you can remove `cx_freeze` as a dependency from [dev-requirements.txt](../dev-requirements.txt). This package is only needed for packaging ClusterRunner binary, and can be ignored for other development purposes.
 
-# Add the next two lines to your shell startup file (.bashrc, .profile, etc.)
-export WORKON_HOME=$HOME/.virtualenvs  # add to .bashrc
-source virtualenvwrapper.sh  # add to .bashrc
+1. Functional tests time out on OSX:
```

Comment:
change this 1. to a 2. :)

I realize markdown will render this as 2, but it bugs me

Reply:
Spoke to @ethomas2 offline. Didn't strongly suggest me to change it. Keeping it as is.